enum.dd

```diff
@@ -4,44 +4,45 @@ $(SPEC_S Enums,
 
 $(GRAMMAR
 $(GNAME EnumDeclaration):
-    $(D enum) $(GLINK EnumTag) $(GLINK EnumBody)
-    $(D enum) $(GLINK EnumBody)
-    $(D enum) $(GLINK EnumTag) $(D :) $(GLINK EnumBaseType) $(GLINK EnumBody)
-    $(D enum) $(D :) $(GLINK EnumBaseType) $(GLINK EnumBody)
-
-$(GNAME EnumTag):
-    $(I Identifier)
+    $(D enum) $(I Identifier) $(GLINK EnumBody)
+    $(D enum) $(I Identifier) $(D :) $(GLINK EnumBaseType) $(GLINK EnumBody)
+    $(AnonymousEnumDeclaration)
 
 $(GNAME EnumBaseType):
-    $(GLINK2 declaration, Type)
+    $(GLINK Type)
 
 $(GNAME EnumBody):
-    $(GLINK EmptyEnumBody)
-    $(GLINK EnumMembersBody)
-
-$(GNAME EmptyEnumBody):
```

Comment:
The `EmptyEnumBody` is used in later documentation. If you remove it, you should also fix the document text.